### PR TITLE
Prepare support python 3.7, 3.8

### PR DESCRIPTION
**Pull request by sh1ng**
I prefer to merge it when tests for python 3.6 are passed as keeping it longer will slowdown CI because of docker cache misses.

* fixed swig wrapper
* LLVM 7.1 as we use quite old LLVM 6.0 that is not supported by python 3.7+
* pylint 

I'm going to add CI configuration for python 3.7 and 3.8 in a separate PR

### CODE REVIEW

File: Dockerfile-build [hunk 1]
```diff
@@ -78,13 +72,23 @@ RUN \
     git checkout tags/v2.4.7-1 && \
     scl enable devtoolset-7 'make CUDA8_GENCODE="-gencode=arch=compute_35,code=sm_35 -gencode=arch=compute_50,code=sm_50 -gencode=arch=compute_60,code=sm_60 -gencode=arch=compute_61,code=sm_61" -j src.build';
 
+#
+# Library versions
+#
+ARG python_version=3.7
```
Comment (pseudotensor) [inline Dockerfile-build:78]: is idea to switch to 3.7 entirely?  Or is this just testing things out?  DAI still supports 3.6 only.

File: Dockerfile-runtime [hunk 2]
```diff
@@ -37,10 +37,12 @@ RUN bash -c 'if [ `arch` = "ppc64le" ]; then yum install -y git boost-devel cmak
 
 RUN mkdir -p /opt/h2oai/h2o4gpu
 
+ARG python_version=3.7
```
Comment (pseudotensor) [inline Dockerfile-runtime:40]: here too, this is overall default now, but probably want it to be controllable which python version to build.

File: ci/Jenkinsfile-ppc64le-cuda10 [hunk 3]
```diff
@@ -30,6 +30,7 @@ def labelruntime = "ibm-power-gpu"
 def doingbenchmark = "0"
 def dobenchmark = "0"
 def doruntime = "1"
+def python = "3.6"
```
Comment (pseudotensor) [inline ci/Jenkinsfile-ppc64le-cuda10:33]: Why 3.6 here?

**Comment by sh1ng** [comment]
I was using default value `3.7` to tests correctness while CI script stays unchanged. Will remove it at all.

**Pull request merged**